Admin/AbstractAdmin.php

```diff
@@ -766,16 +766,28 @@ public function getFilterParameters()
     {
         $parameters = array();
 
+        // if filter persister was configured with `true`, use the default persister (session)
+        if ($this->filterPersister === true) {
+            $this->filterPersister = new SessionFilterPersister($this->request->getSession());
```

Comment:
my aim was to avoid deprecation, but yeah i can do it like it, but the boolean mode will have to be removed